src/components/dialogs/ConfirmDialog.vue

```diff
@@ -138,4 +138,9 @@ export default class ConfirmDialog extends Vue {
 </script>
 <style lang="scss" scoped>
 @import '@/assets/styles/theme.scss';
+
+.dialog-text {
+  color: $gray7 !important;
+  font-size: 1rem;
+}
```

Comment:
What was wrong with the original class (info-text)?

Reply:
It was being overridden, but now that i think about it. Maybe i should override v-card-text in our override. Will try that.

Reply:
Fixed!